### PR TITLE
IA-34 Test properly profile api

### PR DESCRIPTION
**Pull request by olethanh**

### CODE REVIEW

File: iaso/tests/api/test_profiles.py [hunk 1]
```diff
@@ -83,16 +137,126 @@ def test_profile_list_superuser_ok(self):
...
+        user = profile.user
+        self.assertEqual(user.username, data["user_name"])
+        self.assertEqual(user.first_name, data["first_name"])
+
+        self.assertEqual(m.User.objects.filter(username=data["user_name"]).count(), 1)
+        self.assertEqual(profile.account, self.ghi)
+
+        self.assertQuerysetEqual(
+            user.user_permissions.all(),
+            ["<Permission: menupermissions | custom permission support | Formulaires>"],
```
Comment (madewulf) [inline iaso/tests/api/test_profiles.py:245]: I have the feeling this test could break on a change of the __str__(self) of a permission

File: iaso/tests/api/test_profiles.py [hunk 2]
```diff
@@ -8,11 +10,58 @@
...
+        cls.sw_source = sw_source
+        cls.jedi_squad = m.OrgUnitType.objects.create(
+            name="Jedi Squad", short_name="Jds"
+        )
+        cls.jedi_council = m.OrgUnitType.objects.create(
+            name="Jedi Council", short_name="Cnc"
+        )
+        cls.jedi_council.sub_unit_types.add(cls.jedi_squad)
+
+        cls.mock_multipolygon = MultiPolygon(
```
Comment (madewulf) [inline iaso/tests/api/test_profiles.py:42]: I'm curious Why was this needed in a test on profiles?

File: iaso/tests/api/test_profiles.py (hunk 1 above)
Comment (madewulf) [reply iaso/tests/api/test_profiles.py:245]: But that's not a big problem.

**Review by madewulf — APPROVED** [review]

File: iaso/tests/api/test_profiles.py (hunk 2 above)
Comment (olethanh) [reply iaso/tests/api/test_profiles.py:42]: To check if the org unit on the user were correctly set. I copied the code to create orguni from the orgunit api's tests, so it may be more than needed

File: iaso/tests/api/test_profiles.py (hunk 1 above)
Comment (olethanh) [reply iaso/tests/api/test_profiles.py:245]: that's a risk but it make the test much more readable and the change is unlikely to happen and quickly fixed in that case
